indi-gpio/indi_gpio.cpp

```diff
@@ -187,21 +195,36 @@ bool INDIGPIO::Connect()
     }
 
     // Get all lines
+    #ifdef HAVE_LIBGPIOD_V2
     int lines = m_GPIO->get_info().num_lines();
+    #else
+    auto lines = m_GPIO->get_all_lines();
+    #endif
 
     m_InputOffsets.clear();
     m_OutputOffsets.clear();
 
     // Iterate through all lines
+    #ifndef HAVE_LIBGPIOD_V2
     for (int i=1; i<=lines; i++)
```

Comment:
I have no GPIO hardware device, but shouldn't it be counted from 0 to i < lines? That is:

```
// Iterate through all lines
#ifdef HAVE_LIBGPIOD_V2
for (auto i = 0; i < lines; i++)
#else
for (const auto &line : lines)
#endif

```